src/Symfony/EventListener/DenyAccessListener.php

```diff
@@ -85,17 +86,31 @@ private function checkSecurity(Request $request, string $attribute, array $extra
                 $message = $operation->getSecurityMessage();
         }
 
-        if (null === $isGranted) {
```

Comment:
Why is this change needed?

Reply:
Originally it'd jump out of there cause if you have no security string you won't need to do anything in the deny listener ($isGranted is probably also the wrong name, since that variable actually contains the expression for checker).

With the change though you could still have security expressions in the `Link`s so you can't just exit out of there. As you can see the check itself also isn't gone, I just moved it to like 94.